recipes/glfw/all/conanfile.py

```diff
@@ -0,0 +1,99 @@
+import os
+import glob
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration, ConanException
+
+
+class GlfwConan(ConanFile):
+    name = "glfw"
+    description = "GLFW is a free, Open Source, multi-platform library for OpenGL, OpenGL ES and Vulkan" \
+                  "application development. It provides a simple, platform-independent API for creating" \
+                  "windows, contexts and surfaces, reading input, handling events, etc."
+    settings = "os", "arch", "build_type", "compiler"
+    license = "Zlib"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/glfw/glfw"
+    topics = ("conan", "gflw", "opengl", "vulkan", "opengl-es")
+    exports = "LICENSE"
+    generators = "cmake"
+    options = {"shared": [True, False], "fPIC": [True, False]}
+    default_options = {"shared": False, "fPIC": True}
+    _source_subfolder = "source_subfolder"
+    _build_subfolder = "build_subfolder"
+
+    def _check_system_libs(self, libs, install_command):
+        installer = tools.SystemPackageTool()
+        missing_system_libs=[]
+        for package_name in libs:
+            if not installer.installed(package_name):
+                missing_system_libs.append(package_name)
+        if len(missing_system_libs) > 0:
+            raise ConanException(
+                'System libraries missing: {0} not found. You can install using: "sudo {1} {0}"'
+                .format(" ".join(missing_system_libs), install_command))
+
+    def system_requirements(self):
+        if self.settings.os == "Linux":
+            package_tool = tools.SystemPackageTool(conanfile=self, default_mode="verify")
+            libs_name = ""
+            os_info = tools.OSInfo()
+            if os_info.with_apt:
+                libs_name = "libx11-dev libxrandr-dev libxinerama-dev libxkbcommon-dev libxcursor-dev " \
+                            "libxi-dev libglu1-mesa-dev"
+            elif os_info.with_yum:
+                libs_name = "libX11-devel libXrandr-devel libXinerama-devel libxkbcommon-devel " \
+                            "libXcursor-devel libXi-devel mesa-libGL-devel"
+            elif os_info.with_pacman:
+                libs_name = "libx11 libxrandr libxinerama libxkbcommon-x11 libxcursor libxi libglvnd "
+            else:
+                self.output.warn("Could not find any package manager, please install x11, xrandr, xinerama," \
+                                 "xkb, xcursor, xi and mesa libraries if not already installed.")
+                return
+            package_tool.install(update=True, packages=libs_name)
```

Comment:
[doc](https://docs.conan.io/en/latest/reference/conanfile/methods.html#method-system-requirements) states that `To install different packages, one call to install() per package is necessary.`. Is it outdated?

Reply:
I think that refers to the fact that a parameter is an actual list: 
`packages=["pkg1", "pkg1-alternative1", "pkg1-alternative2"]` 
but if the parameter is a space-separated string it will install the different packages 
`packages="pkg1, pkg2, pkg3"`

Reply:
I agree with Eric, adding all variables in one string is not documented.